### DISCUSSION TITLE
Differences from LOLINTERNETZ/vscodeoffline

### DISCUSSION THREAD

**Discussion by DataOps7** [coder/code-marketplace#105] [Q&A]
What does this repo offer that the `LOLINTERNETZ/vscodeoffline` repo doesn't?

**Comment by code-asher** [comment]
This is the first I have seen it.  Looks cool.  From my brief glance, it seems to implement more than we do, except paging?  The mirror/sync seems useful as well, although syncing Microsoft's marketplace may be legally dubious.  With our project, you have to add the extensions you want explicitly via their `.vsix` files (either local or from a URL).  We have Artifactory support, not sure what `vscodeoffline` does (maybe just uses the local filesystem?).

**Comment by DataOps7** [comment]
Thank you so much for your time!
It seems like they only support local filesystem, but also support VSCode binaries and other cool features.
I'm trying to understand what would be the best solution as I'm working with coder, code-server and Artifactory in an air-gapped network.
I want code-server to be able to install new extensions seamlessly but also allow local users using VSCode / VSCodium to access that marketplace.

**Comment by code-asher** [comment]
Ah yeah the Artifactory requirement might force you to use this repo, although like you mentioned we lack stuff like binaries, and no extension signing yet which might cause issues because we had reports signing is now required by the latest VS Code.
